test/parallel/test-diagnostics-channel-tracing-channel-args-types.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+require('../common');
+const dc = require('diagnostics_channel');
+const assert = require('assert');
+
+let channel;
+
+// tracingChannel creating with name
+channel = dc.tracingChannel('test');
+assert.match(channel.start.name, /:test:/);
```

Comment:
```suggestion
assert.strictEqual(channel.start.name, 'tracing:test:start');
```
And maybe add asserts for all channels here.

Reply:
Resolved.